includes/definitions/discovery/dlink.yaml

```diff
@@ -0,0 +1,12 @@
+mib: EQUIPMENT-MIB
+
+modules:
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: swTemperatureTable
+                    value: swTemperatureCurrent
+                    num_oid: .1.3.6.1.4.1.171.12.11.1.8.1.2.
+                    descr: Current Sys Temp
```

Comment:
If you get more than one result back here then the descr will not be unique so just change it to: `descr: 'Current Sys Temp {{ $index }}'`